presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -915,12 +923,14 @@ void PrestoServer::updateAnnouncerDetails() {
 }
 
 void PrestoServer::addServerPeriodicTasks() {
+  const auto* systemConfig = SystemConfig::instance();
```

Comment:
I think you need to capture this pointer by copy in the below lambda. Or you can simply put the SystemConfig::instance() call inside lambda. Yeah let's just move it inside lambda. There is not a strong need of having it outside.